specs/engage-digital_openapi3.yaml

```diff
@@ -884,6 +884,48 @@ paths:
             items:
               type: string
             type: array
+        - description: Signatures
+          in: query
+          name: signatures
+          required: false
+          schema:
+            type: array
+            items:
+              type: object
+            example: [{ body: 'Hello World', language: 'en' }]

```

Comment:
Fixed : 

![Screenshot 2022-09-05 at 13 33 38](https://user-images.githubusercontent.com/8244522/188440889-0fe50182-61ae-4100-adf2-c18731d1134a.png)

I try to display it on the left section but it's not supported yet -> There is an opening issue for it : https://github.com/swagger-api/swagger-ui/issues/4581